### PR TITLE
V6 queued move plugins

### PR DESCRIPTION
**Pull request by Bind**

### CODE REVIEW

File: content/productivity/distribute-silver/plugin.js [hunk 1]
```diff
@@ -110,7 +110,7 @@ class Plugin {
   }
 }
 
-plugin.register(new Plugin());
+export default Plugin;
```
Comment (phated) [inline content/productivity/distribute-silver/plugin.js:113]: Oh weird, was this being exported twice? Let's remove one 😛 
```suggestion
```

File: content/productivity/distribute-silver/plugin.js [hunk 2]
```diff
@@ -110,7 +110,6 @@ class Plugin {
   }
 }
 
-export default Plugin;
```
Comment (Bind) [inline content/productivity/distribute-silver/plugin.js]: woops!

**Comment by phated** [comment]
Oops, needs rebase.

**Review by phated — APPROVED** [review]
Awesome ❤️

**Pull request merged**